ui_framework/src/components/toast/_toast.scss

```diff
@@ -81,5 +90,14 @@ $toastTypes: (
   }
 
 .kuiToastHeader--withBody {
-  margin-bottom: $kuiVerticalRhythmS;
+  margin-bottom: $kuiSizeM;
```

Comment:
We should probably just remove this class (and remove references to it from the React component), and use the same pattern you used here: https://github.com/elastic/kibana/pull/13435#discussion_r132748431